src/curated-corpus/components/ExistingProspectCard/ExistingProspectCard.tsx

```diff
@@ -131,7 +132,7 @@ export const ExistingProspectCard: React.FC<ExistingProspectCardProps> = (
                 </Typography>
               </Link>
             </Grid>
-            <Grid item xs={1}>
+            <Grid item xs={1} sx={{ textAlign: 'center' }}>
```

Comment:
This improves positioning of the "Dismiss" cross button on the existing prospect card.